src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java

```diff
@@ -257,6 +257,13 @@ public MockResponse dispatch(RecordedRequest request) {
         server.setDispatcher(dispatcher);
     }
 
+    private Map<String, Object> createRequestBody(String elementType, List<String> substationsIds) {
+        Map<String, Object> requestBody = new HashMap<>();
+        requestBody.put("elementType", elementType);
+        requestBody.put("substationsIds", substationsIds);
+        return requestBody;
+    }
```

Comment:
make this method use "mapper.writeValueAsString" and return a string so you don't have to use "mapper.writeValueAsString" everywhere ?